### DISCUSSION TITLE
What is the difference between setting the value 1 and 2 to CURLOPT_SSL_VERIFYHOST

### DISCUSSION THREAD

**Discussion by ganeshkamath89** [curl/curl#6451] [Q&A] [answered]
In our code, the VerifyHost value is set to 2 for HTTPS.

We have observed that when our client is tested against one particular project it is failing with "SSL Connection error" with libCurl. But this error is not observed in an older version of our client which used WinHTTP. On other platforms however both WinHTTP and Curl succeed.

I wanted to know what happens if I change CURLOPT_SSL_VERIFYHOST from 2 to 1. What is the difference in behaviour for these two values?

**Comment by jzakrzewski** [comment, marked as answer]
I think the documentation explains that: https://curl.se/libcurl/c/CURLOPT_SSL_VERIFYHOST.html

**Comment by ganeshkamath89** [comment]
So as per the documentation, does it mean it will not affect me if I am using Curl 7.74 and pass the value 1, will it still behave the same way?

**Reply by bagder** [reply]
yes

**Reply by ganeshkamath89** [reply]
Thanks a lot
